### PR TITLE
Sort tables in .toml files

### PR DESCRIPTION
**Pull request by brooksprumo**
#### Problem

The tables in .toml are not sorted. Did this bother anyone else, or just me?

#### Summary of Changes

I used `cargo sort --workspace` to sort the tables 😃

### CODE REVIEW

**Comment by t-nelson** [comment]
bonus points if you enforce it in `ci/test-sanity.sh`

File: account-decoder/Cargo.toml [hunk 1]
```diff
@@ -10,11 +10,11 @@ license = "Apache-2.0"
 edition = "2021"
 
 [dependencies]
+Inflector = "0.11.4"
```
Comment (CriesofCarrots) [inline account-decoder/Cargo.toml:13]: Hahaha :eyetwitch:

Comment (t-nelson) [reply account-decoder/Cargo.toml:13]: I too prefer that it was pictographicly sorted next to lowercase ell before

**Comment by brooksprumo** [comment]
> bonus points if you enforce it in `ci/test-sanity.sh`

I agree. Created an issue to track it: https://github.com/solana-labs/solana/issues/23608.

**Comment by brooksprumo** [comment]
@CriesofCarrots @t-nelson Sorry for the additional ping here... Since this PR touches a lot of files (and Cargo.toml files especially), there's a high chance of this PR ending up with conflicts. It would be great to merge as soon as possible (assuming CI completes successfully). I'm happy to address any concerns/requests, and if not, could I get an "approve"?

Also, I looked into the sorting, to put capital letters next to lowercase... Turns out no luck (at least with the existing config options). If this is a deal-breaker, that's also fine and I can close the PR.

**Review by CriesofCarrots — APPROVED** [review]
> Also, I looked into the sorting, to put capital letters next to lowercase... Turns out no luck (at least with the existing config options). If this is a deal-breaker, that's also fine and I can close the PR.

It's not a deal-breaker, I'll just have to never work in account-decoder again :trollface: 
I am actually a little surprised that `cargo sort` doesn't have different options for that.

Anyway, thanks for doing this and introducing me to `cargo sort` in the first place!

**Pull request merged**